stsci.skypac/meta.yaml

```diff
@@ -27,7 +28,7 @@ requirements:
     - numpy {{ numpy }}
     - python {{ python }}
     run:
-    - astropy >=1.1
+    - astropy >=3.1
```

Comment:
Strictly speaking, you can probably support `astropy >= 3` (which would include 3.0.x) unless you use `astropy` feature that only exists in 3.1 like APE14.

Reply:
Yes, I need `3.1` (because of APE 14) - see `_naxis1` issues.